link_shortener/templates/delete_link.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <br>
+    <h1>Are you sure you know what you're doing?</h1>
+        <h2>Like really?</h2>
+    <h3>If so, then click on to kill it, else..ruuuuun</h3>
+    <br>
+    <br>
+    <br>
+    <a href="javascript:history.back()"><button>Take me back!!</button></a>
+    <a href="/delete/proceed/{{ payload }}"><button>Yo, kill it!</button></a>
+{% endblock %}
```

Comment:
Možná sem přihoď tu newlinu ať to neřve. Většina editorů je doplňuje automaticky, možná to zkus ve svym taky nastavit.

Reply:
Tady nerozumim :(

Reply:
Jak je tam to červený kolečko, to brečí že chce prázdný řádek na konci

Reply:
PyCharm nerkici, ale ju chapem uz :)